test/powershell/engine/Jobs.Tests.ps1

```diff
@@ -0,0 +1,17 @@
+Describe 'Basic Job Tests' -Tags 'CI' {
+
+    $job = Start-Job {1}
+
+    It 'Test job creation' {
+        $job | should not be $null
+    }
+
+    It 'Test job State' {
+        Wait-Job $job -Timeout 5
```

Comment:
5 seconds might not be enough time to wait on a heavily loaded system, especially with debug binaries that aren't pre-jit compiled.